java/server/src/org/openqa/selenium/grid/docker/DockerOptions.java

```diff
@@ -124,7 +124,7 @@ public void configure(HttpClient.Factory clientFactory, LocalNode.Builder node)
       for (int i = 0; i < maxContainerCount; i++) {
         node.add(caps, new DockerSessionFactory(clientFactory, docker, image, caps));
       }
-      LOG.info(String.format(
+      LOG.finest(String.format(
```

Comment:
This change prevents a user understanding how their server is configured. Best to leave at `info` level.